include/valkey/cluster.h

```diff
@@ -156,33 +144,49 @@ typedef struct valkeyClusterNodeIterator {
     char opaque_data[VALKEY_NODE_ITERATOR_SIZE];
 } valkeyClusterNodeIterator;
 
-/*
- * Synchronous API
- */
+/* --- Configuration options --- */
+
+/* Enable slotmap updates using the command CLUSTER SLOTS.
+ * Default is the CLUSTER NODES command. */
+#define VALKEY_OPT_USE_CLUSTER_SLOTS 0x1000
```

Comment:
Should we consider changing the default to SLOTS? It's recommended in Valkey.... But NODES is better for clusters with highly fragmented slot ranges.

Reply:
I like the idea of changing the default to SLOTS, specially if it's recommended.
We could add an option in the future that let the client decide and predicted which command is most efficient.

Reply:
I think we should support both to make sure users can migrate without any changed characteristics or behaviour, but we can change the default. We can have a flag for VALKEY_OPT_USE_CLUSTER_NODES instead.